login/token_src.go

```diff
@@ -126,25 +126,31 @@ func (s *OIDCTokenSource) getOIDCConfigWithRedirectURL(redirectURL string) oidc.
 	return oidcConfig
 }
 
+// OIDCToken is the same as OIDCTokenCtx, except it uses context from itself.
```

Comment:
Add warning `Do not user per request timeouts in stuct ctx` .. maybe in constructor?

Reply:
done

Reply:
sorry - now with new changes we always use request ones, so we can kill these comments